tests/test_mysql.py

```diff
@@ -0,0 +1,71 @@
+import mysql.connector
+from mysql.connector import Error
+
+db_config = {
+    "host": "localhost",
+    "user": "root",
+}
+
+create_stats_container_table = """
+CREATE TABLE stats_container (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    key VARCHAR(255) NOT NULL
+);
+"""
+
+create_statsworkers_table = """
+CREATE TABLE stats_worker (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    stats_container INT NOT NULL,
+    bad INT,
+    count INT,
+    FOREIGN KEY (stats_container) REFERENCES stats_container(id)
+);
+"""
+
+create_statsbins_table = """
+CREATE TABLE stats_bin (
+    id INT AUTO_INCREMENT PRIMARY KEY,
+    stats_worker_id INT NOT NULL,
+    msize INT,
+    val INT,
+    FOREIGN KEY (stats_worker_id) REFERENCES stats_worker(id)
+);
+"""
+
+def setup_database():
+    connection = mysql.connector.connect(**db_config)
```

Comment:
going forward we may have more complex queries, where an ORM could be easier to work with than raw SQL, which can be pretty error prone when writing it by hand. I can change this test file to use sqlalchey so that we pick one and not use both

Reply:
I know what they are for.   I've used them a lot.   In my experience You should avoid them unless you need them.   Right now we don't.  ideally we shouldn't be using a database and we should be using our own abstractions and not someone else's

Reply:
ok  I will yank it out and do raw sql

Reply:
We can introduce an ORM if we really need it.  I'm hoping we never access the database from this service and eventually get rid of accessing it completely.